### PR TITLE
vis_keypoint --> vis_point

### PR DESCRIPTION
**Pull request by yuyu2172**
This change is based on the new naming convention (`keypoint, kp_mask --> point, mask`)  https://github.com/chainer/chainercv/issues/507

### CODE REVIEW

File: chainercv/visualizations/vis_point.py [hunk 1]
```diff
@@ -4,32 +4,32 @@
...
+            the number of points.
             The second axis corresponds to :math:`y` and :math:`x` coordinates
-            of the keypoint.
-        kp_mask (~numpy.ndarray, optional): A boolean array whose shape is
-            :math:`(K,)`. If :math:`i` th index is :obj:`True`, the
-            :math:`i` th keypoint is not displayed. If not specified,
-            all keypoints in :obj:`keypoint` will be displayed.
+            of the points.
+        mask (~numpy.ndarray, optional): A boolean array whose shape is
+            :math:`(P,)`. If :math:`i` th index is :obj:`True`, the
```
Comment (Hakuyume) [inline chainercv/visualizations/vis_point.py]: `:math:i th index` -> `:math:i th element`

Comment (yuyu2172) [reply chainercv/visualizations/vis_point.py]: Thanks

File: chainercv/visualizations/vis_point.py [hunk 2]
```diff
@@ -4,32 +4,32 @@
...
+            :math:`(P, 2)`, where :math:`P` is
+            the number of points.
             The second axis corresponds to :math:`y` and :math:`x` coordinates
-            of the keypoint.
-        kp_mask (~numpy.ndarray, optional): A boolean array whose shape is
-            :math:`(K,)`. If :math:`i` th index is :obj:`True`, the
-            :math:`i` th keypoint is not displayed. If not specified,
-            all keypoints in :obj:`keypoint` will be displayed.
+            of the points.
+        mask (~numpy.ndarray, optional): A boolean array whose shape is
```
Comment (Hakuyume) [inline chainercv/visualizations/vis_point.py]: `optional` in the type description may be against to our convention.
c.f. https://github.com/chainer/chainercv/blob/master/chainercv/visualizations/vis_bbox.py#L27-L29

File: chainercv/visualizations/vis_point.py [hunk 3]
```diff
@@ -42,18 +42,18 @@ def vis_keypoint(img, keypoint, kp_mask=None, ax=None):
...
 
-    if kp_mask is None:
-        kp_mask = np.ones((n_kp,), dtype=np.bool)
+    if mask is None:
+        mask = np.ones((n_point,), dtype=np.bool)
 
     cm = plot.get_cmap('gist_rainbow')
 
-    colors = [cm(1. * i / n_kp) for i in six.moves.range(n_kp)]
+    colors = [cm(1. * i / n_point) for i in six.moves.range(n_point)]
```
Comment (Hakuyume) [inline chainercv/visualizations/vis_point.py]: You can write `cm(i / n_point)` with `from __feature__ import division`.

File: chainercv/visualizations/vis_point.py [hunk 4]
```diff
@@ -1,35 +1,37 @@
...
-        kp_mask (~numpy.ndarray, optional): A boolean array whose shape is
-            :math:`(K,)`. If :math:`i` th index is :obj:`True`, the
-            :math:`i` th keypoint is not displayed. If not specified,
-            all keypoints in :obj:`keypoint` will be displayed.
+            of the points.
+        mask (~numpy.ndarray): A boolean array whose shape is
+            :math:`(P,)`. If :math:`i` th element is :obj:`True`, the
+            :math:`i` th point is not displayed. If not specified,
+            all points in :obj:`point` will be displayed.
         ax (matplotlib.axes.Axes, optional): If provided, plot on this axis.
```
Comment (Hakuyume) [inline chainercv/visualizations/vis_point.py]: `optional`

Comment (yuyu2172) [reply chainercv/visualizations/vis_point.py]: my bad

**Pull request merged**